test/integration/google_sso_test.rb

```diff
@@ -1,26 +1,28 @@
 require 'test_helper'
```

Comment:
Do we have a followup ticket to investigate & fix this?

Reply:
Racked in #1311